devicetypes/smartthings/zigbee-motion-sensor-light.src/led-cpx-light.groovy

```diff
@@ -0,0 +1,168 @@
+				attributeState "level", action: "switch level.setLevel"
+			}
+		}
+		
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label: "", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+		
+		controlTile("colorTempSliderControl", "device.colorTemperature", "slider", width: 4, height: 2, inactiveLabel: false, range: "(2700..6500)") {
+			state "colorTemperature", action: "color temperature.setColorTemperature"
+		}
+		
+		main(["switch"])
+		details(["switch", "switchLevel", "colorTempSliderControl", "refresh"])
+	}
+}
+
+private getMOTION_CLUSTER() { 0x0406 }
+private getMOTION_STATUS_ATTRIBUTE() { 0x0000 }
+private getON_OFF_CLUSTER_VALUE() { 0x0006 }
+private getCONFIGURE_REPORTING_RESPONSE() { 0x07 }
+
+def parse(String description) {
+	def event = zigbee.getEvent(description)
+	
+	if (event) {
+		if (!(event.name == "level" && event.value == 0)) {
+			sendEvent(event)
+		}
+	} else {
+		def cluster = zigbee.parse(description)
+		
+		if (cluster && cluster.clusterId == ON_OFF_CLUSTER && cluster.command == CONFIGURE_REPORTING_RESPONSE) {
+			if (cluster.data[0] == 0x00) {
+				sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+			}
+		} else {
+			def zigbeeMap = zigbee.parseDescriptionAsMap(description)
+			
+			if (zigbeeMap.clusterInt ==  MOTION_CLUSTER && zigbeeMap.attrInt == MOTION_STATUS_ATTRIBUTE) {
+				def childDevice = getChildDevices()?.find {
+					it.device.deviceNetworkId == "${device.deviceNetworkId}:1" 
+				}
+				zigbeeMap.value = zigbeeMap.value.endsWith("01") ? "active" : "inactive"			
+				zigbeeMap.name = "motion"
+				childDevice.sendEvent(zigbeeMap)
+			}
+		}
+	}
+}
+
+def off() {
+	zigbee.off()
+}
+
+def on() {
+	zigbee.on()
+}
+
+def setLevel(value, rate=null) {
+	zigbee.setLevel(value)
+}
+
+def configure() {
+	zigbee.configureReporting(MOTION_CLUSTER, MOTION_STATUS_ATTRIBUTE, 0x18, 30, 600, null) +
+		zigbee.onOffConfig() +
+		zigbee.levelConfig() +
+		refresh()
+}
+
+def updated() {
+	if (!childDevices) {
+		addChildSensor()
+	}
+}
+
+def ping() {
+	return zigbee.levelRefresh()
+}
+
+def refresh() {
+	zigbee.readAttribute(MOTION_CLUSTER, MOTION_STATUS_ATTRIBUTE) +
+		zigbee.onOffRefresh() +
+		zigbee.levelRefresh() +
+		zigbee.colorTemperatureRefresh()
+}
+
+def setColorTemperature(value) {
+	value = value as Integer
+	
+	zigbee.setColorTemperature(value) +
+		zigbee.on() +
+		zigbee.colorTemperatureRefresh()
+}
+
+def installed() {
+	addChildSensor()
+}
+
+def addChildSensor() {
+	def componentLabel
+	def childDevice
+	
+	if (device.displayName.endsWith(' Light') || device.displayName.endsWith(' light')) {
+		componentLabel = "${device.displayName[0..-6]} Motion sensor"
+	} else {
+		componentLabel = "$device.displayName Motion sensor"
+	}
+	
+	if (childDevice != null) {
+		childDevice.sendEvent(name: "motion", value: "inactive")
+	}
```

Comment:
As @PKacprowiczS said, this will always be null if you call it here. I imagine you want to move this inside the `try` block

Reply:
Yes, I changed it.